Doc/c-api/object.rst

```diff
@@ -492,6 +492,13 @@ Object Protocol
    on failure.  This is equivalent to the Python statement ``del o[key]``.
 
 
+.. c:function:: int PyObject_DelItemString(PyObject *o, const char *key)
+
+   This is the same as :c:func:`PyObject_DelItem`, but *key* is
```

Comment:
This is the canonical[^1] way to write it (`git grep -i "similar to" Doc/c-api` for inspiration):

```suggestion
   Similar to :c:func:`PyObject_DelItem`, but *key* is
```

[^1]: _"This is the same as"_ is sometimes used, but _"Similar to [...], but"_ is a more common, and also more concise.